openpower/patches/vesnin-patches/skiboot/skiboot-1001-core-direct-controls.c-Add-sreset-support-for-P8.patch

```diff
@@ -0,0 +1,52 @@
+From 016b60d166744b1727271a4e230ccb335112bc6b Mon Sep 17 00:00:00 2001
+From: Maxim Polyakov <m.polyakov@yadro.com>
+Date: Fri, 18 Jan 2019 17:49:37 +0300
+Subject: [PATCH] core/direct-controls.c: Add sreset support for P8
```

Comment:
`Subject: [PATCH] core/direct-controls: Add sreset support for P8`

Reply:
Fixed